src/undo/rollback.py

```
@@ -25,14 +27,23 @@ class RollbackExecutor:
     to undo or redo file operations.
     """
 
-    def __init__(self, validator: OperationValidator | None = None):
+    def __init__(
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)